### PR TITLE
Photon ui tweaks - blue border fix

### PR DESCRIPTION
**Pull request by marvinkome**
Fixes Issue: #6180

### Summary of Changes
* make border thicker when focused

### CODE REVIEW

File: src/components/shared/SearchInput.css [hunk 1]
```diff
@@ -7,7 +7,8 @@
 }
 
 .search-shadow.focused {
-  box-shadow: var(--theme-focus-box-shadow-textbox);
+  box-shadow: 0 0 0 1px rgb(101, 183, 255);
+  border: 0.5px solid rgb(101, 183, 255);
 }
```
[the remark this answers is not in the document]
Comment (marvinkome) [reply src/components/shared/SearchInput.css]: screenshot from 2018-05-27 14-36-13

Thats how it looks on dark theme

[the remark this answers is not in the document]
Comment (jasonLaster) [reply src/components/shared/SearchInput.css]: thanks. can you also share a light theme screenshot too?

[the remark this answers is not in the document]
Comment (marvinkome) [reply src/components/shared/SearchInput.css]: dbgrhtml
screenshot from 2018-05-28 15-27-47

I sent the wrong screenshot. These are the correct screenshots. Light and dark

[the remark this answers is not in the document]
Comment (darkwing) [reply src/components/shared/SearchInput.css]: Where did you get this value from?

[the remark this answers is not in the document]
Comment (marvinkome) [reply src/components/shared/SearchInput.css]: From my brain. I just used a color picker to get the previous color and thickened it

**Comment by darkwing** [comment]
@violasong Do you care to weigh in on this?  Otherwise we can merge.  Thanks!

**Comment by violasong** [comment]
Hi @marvinkome, thanks for working on this. The problem with the styling of this border was actually that it was the wrong color, not that it was too thin. It should match Console's focus rings, so `border: 1px solid var(--blue-50);` in both light and dark mode. Sorry about the confusion.

**Comment by violasong** [comment]
One other thing I noticed from the screenshots is that it looks like the selected row style is a different blue from before. It should match the selected row color of the inspector (different values for dark and light mode). Thanks!

**Comment by darkwing** [comment]
We can update the selected row style in another PR.  Feel free to update per @violasong's color -- thanks @marvinkome !

**Comment by marvinkome** [comment]
@violasong I don't really understand you. Initially it was just a box-shadow. I add a `0.5px` border to increase the thickness. Are you saying I should change the color of the box-shadow instead it Change the color for both? 

Also the selected row color. Would be a new change. I didn't make any changes there.

**Comment by violasong** [comment]
@marvinkome I meant that this focus ring should match the console focus rings implementation (I think they just use a 1px transparent border on their input field which turns Blue 50 on focus, no box-shadow needed).

Ah, good to know about the selected row color - maybe it just looks different in this screenshot than on my screen.

**Comment by darkwing** [comment]
@nchevobbe Can you illuminate what the console focus ring style is so we can wrap this one up?  Thank you!

**Comment by marvinkome** [comment]
@darkwing 'm sorry I haven't fixed this yet, I've been offline for the past 3 days, I'll submit a new pr today. Thank you.

**Comment by darkwing** [comment]
I've used the Browser Toolbox to discover the correct color and settings of the console:

consoleborder

I've applied them in the latest commit.  Additionally, I looked at the style for selected rows in the inspector and they do match:

matches2

If tests pass this is ready to go!

**Comment by violasong** [comment]
Thanks @darkwing for the fix and for verifying the selected row color!

**Pull request merged**